gnovm/cmd/gno/mod_test.go

```diff
@@ -189,20 +190,21 @@ valid.gno
 			simulateExternalRepo: true,
 			stdoutShouldBe:       ``,
 		},
-		{
-			args:                 []string{"mod", "graph"},
-			testDir:              "../../tests/integ/valid2",
-			simulateExternalRepo: true,
-			stdoutShouldBe: `gno.land/p/integ/valid gno.land/p/demo/avl
-`,
-		},
-		{
-			args:                 []string{"mod", "graph"},
-			testDir:              "../../tests/integ/require_remote_module",
-			simulateExternalRepo: true,
-			stdoutShouldBe: `gno.land/tests/importavl gno.land/p/demo/avl
-`,
-		},
+		// TODO: gno mod graph should disappear?
```

Comment:
Definitely not; it should be rewritten to avoid using the replace directives and instead utilize lazy dependency computation (already existing). It will also be significantly improved once we have the new package downloader.

```suggestion
		// TODO: port gno mod graph to the new dependency discovery system.
```